controllers/storage/minio.go

```diff
@@ -99,7 +118,12 @@ func minioReadyStatus(properties *lcm.Properties) *lcm.CRStatus {
 }
 
 func (m *MinIOReconciler) Delete() (*lcm.CRStatus, error) {
```

Comment:
Maybe we don't need to delete the function? By setting the OwnerReference to let the controller-manager perform GC.

Reply:
when the storage type switched, such as the type of inCluster change to external, we will use it to delete minio  CR.